### ISSUE TITLE
Is there any ways to load aerial when use cmd "AerialToggle"

### ISSUE THREAD

**Issue by NEX-S** [stevearc/aerial.nvim#117]
It seems we must load aerial in my lsp-config's on_attach function, i want to load aerial when i exec "AerialToggle", is there any ways can reach this? ( i thought it must use some callback functions to do this.... but i dont know how to write it ...)

**Comment by stevearc** [comment]
At the moment there's not a good way to do that. I have some ideas that I think I could adapt from a new plugin I'm working on, but I'm very busy for the next two weeks so I won't be able to look into this anytime soon.

**Comment by stevearc** [comment]
I've recently completed a large refactor that takes advantage of Neovim 0.8's features. As part of this, I was able to remove the need for the lsp `on_attach` (we can use the LspAttach autocmd now) and have added the ability for aerial to lazy-load itself (meaning calling `require("aerial").setup()` should be extremely fast). In most cases, the lazy loading behavior is enabled by default. Check the documentation for the "lazy_load" config option for details.
